app/assets/stylesheets/shared/filter_dropdown.scss

```diff
@@ -29,12 +29,24 @@
       position: relative;
       width: 100%;
 
+      &.folders {
```

Comment:
can you rename this class in protocol partial? And add to only protocol CSS, because this checkbox is unique for this view.

Reply:
<img width="488" alt="Screenshot 2023-02-28 at 15 56 42" src="https://user-images.githubusercontent.com/47317017/221891360-7d50864e-0dca-47bc-a933-c1a30f46260e.png">
So it will not `folder`, but `has-draft`. I forgot to update.